x-pack/plugins/enterprise_search/public/applications/app_search/app_logic.test.ts

```diff
@@ -21,6 +21,21 @@ describe('AppLogic', () => {
     jest.clearAllMocks();
   });
 
+  const EXPECTED_BASE_RESULTS = {
```

Comment:
NIT: Is there any reason not to call this `DEFAULT_VALUES` to match other places, for consistency?